src/components/LeaderBoard.js

```diff
@@ -163,139 +53,120 @@ function LeaderBoard(props) {
-          className={`${classes.rectangle69} ${classes.colorPalette2} ${classes.clickable}`}
-          align="center"
-          left="220px"
-          top="-100px"
-        >
-          <Avatar className={classes.eclipse}>
-            <Avatar
-              className={classes.profileImage}
-              alt="alt name"
-              src={countries[0].flag}
-            />
-          </Avatar>
-          <Box id="inner2" className={classes.rectangle81}>
-            <Box className={classes.planter}>Planter</Box>
-            <Box className={classes.content}>{countries[0].name}</Box>
-            <Box className={classes.rating}>
-              Planted
-              <br></br>
-              {countries[0].planted}
-            </Box>
-          </Box>
-        </Box>
-      </Box>
-      <Box>
-        <Box
-          onClick={() => handleCountryClick(countries[2].id)}
-          className={`${classes.rectangle69} ${classes.colorPalette3} ${classes.clickable}`}
-          align="center"
-          left="440px"
-          top="-230px"
-          background="linear-gradient(139.31deg, rgba(71, 75, 79, 0.2) 0%, rgba(97, 137, 47, 0.9) 77.54%);"
-        >
-          <Avatar className={classes.eclipse}>
-            <Avatar
-              className={classes.profileImage}
-              alt="alt name"
-              src={countries[2].flag}
-            />
-          </Avatar>
-          <Box id="inner3" className={classes.rectangle81}>
-            <Box className={classes.planter}>Planter</Box>
-            <Box className={classes.content}>{countries[2].name}</Box>
-            <Box className={classes.rating}>
-              Planted
-              <br></br>
-              {countries[2].planted}
-            </Box>
-          </Box>
-        </Box>
+          <Grid item xs={1}>
+            {/* Empty space */}
+          </Grid>
+          <Grid item xs={1}>
+            <Typography variant="h5" className={classes.customText}>
+              #
+            </Typography>
+          </Grid>
+          <Grid item xs={7}>
+            <Typography variant="h5" className={classes.customText}>
+              COUNTRY
+            </Typography>
+          </Grid>
+          <Grid item xs={7}>
+            <Typography variant="h5" className={classes.customText}>
+              TREES PLANTED <ParkTwoToneIcon style={{ color: 'green' }} />
+            </Typography>
+          </Grid>
+        </Grid>
       </Box>
-      {countries &&
-        countries.slice(3).map((country, index) => (
+
+      {countries ? (
+        countries.map((country, index) => (
           <Box
             key={country.id}
-            onClick={() => handleCountryClick(countries[1].id)}
-            className={`${classes.rectangle72}  ${classes.clickable}`}
+            sx={{
+              flexGrow: 1,
+              borderRadius: 16,
+              borderColor: 'grey.500',
+              boxShadow: 5,
+            }}
+            margin={2}
+            borderColor={'beige.500'}
```

Comment:
`borderColor` is defined twice here. Because this component already has the `sx` prop I think it is better to be consistent and keep all styles for this component inside the sx object. It's also important to be aware of the unique differences between the sx prop and makeStyles. You can read about those here https://mui.com/system/the-sx-prop/ specifically be aware of the borderRadius difference 

```
The borderRadius properties multiples the value it receives by the theme.shape.borderRadius value (the default for the value is 4px).
```